deviceModeInit.js

```diff
@@ -95,6 +96,117 @@ var rudderTracking = (function () {
       .catch((error) => {
         console.log("Error occurred while updating cart:", error);
       });
+
```

Comment:
Can we add the JSDOC description with what the function(s) is doing?

Reply:
Its fine it will make script heavier later on when we move this things to typescript we can add JS doc

Reply:
Yes got it.